code/modules/xenoarchaeology/xenoartifact_labeler.dm

```diff
@@ -0,0 +1,226 @@
+/obj/item/xenoartifact_labeler
+	name = "Xenoartifact Labeler"
+	icon = 'icons/obj/xenoarchaeology/xenoartifact_tech.dmi'
+	icon_state = "xenoartifact_labeler"
+	desc = "A tool scientists use to label their alien bombs."
+	throw_speed = 3
+	throw_range = 5
+	w_class = WEIGHT_CLASS_TINY
+
+	var/list/activator = list() //Checked trait
+	var/list/activator_traits = list() //Display names
+
+	var/list/minor_trait = list()
+	var/list/minor_traits = list()
+
+	var/list/major_trait = list()
+	var/list/major_traits = list()
+
+	var/list/malfunction = list()
+	var/list/malfunction_list = list()  
```

Comment:
Why are there two lists that seemingly do the same thing?

Reply:
One is all the traits, one is the selected traits. It's all seperate for grouping purposes.

Reply:
then name the vars better to indicate that difference